### PR TITLE
Changed package structure and group id

### PR DESCRIPTION
**Pull request by whiskeysierra**
See #55

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ To use SProcWrapper, add the following lines to your pom.xml:
 
 ```xml
 <dependency>
-    <groupId>de.zalando</groupId>
+    <groupId>org.zalando</groupId>
     <artifactId>zalando-sprocwrapper</artifactId>
```
Comment (whiskeysierra) [inline README.md]: Should we rename it to `sprocwrapper`?

File: src/main/java/org/zalando/sprocwrapper/proxy/StoredProcedure.java [hunk 2]
```diff
@@ -82,7 +75,7 @@
     private final boolean searchShards;
     private final boolean parallel;
     private final boolean readOnly;
-    private final WriteTransaction writeTransaction;
+    private final SProcService.WriteTransaction writeTransaction;
```
Comment (vadeg) [inline src/main/java/org/zalando/sprocwrapper/proxy/StoredProcedure.java]: Some weird refactoring. Should it be `WriteTransaction` with 
```
import de.zalando.sprocwrapper.SProcService.WriteTransaction;
```
?

File: README.md (hunk 1 above)
Comment (whiskeysierra) [reply README.md]: We can change that as well. Github will manage a redirect, so we don't loose anything.

Comment (whiskeysierra) [reply README.md]: @Jan-M Objections?

**Comment by whiskeysierra** [comment]
> Imports order was changed in many places. What kind of import configuration have you used?

Standard IntelliJ IDEA settings.

File: src/main/java/org/zalando/sprocwrapper/proxy/StoredProcedure.java (hunk 2 above)
Comment (whiskeysierra) [reply src/main/java/org/zalando/sprocwrapper/proxy/StoredProcedure.java]: Looks like a side effect of *organize imports*. I'll take a look.

File: README.md (hunk 1 above)
Comment (Jan-M) [reply README.md]: i have no objections, given there is some rule of using .org for open source libraries.

File: src/main/java/org/zalando/sprocwrapper/proxy/StoredProcedure.java (hunk 2 above)
Comment (vadeg) [reply src/main/java/org/zalando/sprocwrapper/proxy/StoredProcedure.java]: Any updates on this?

**Pull request closed without merging**